homeassistant/components/vicare/const.py

```diff
@@ -34,6 +34,41 @@
 }
 
 
+class Program(enum.StrEnum):
```

Comment:
I used that before but there are also programs for cooling (not used till now).

Reply:
Then that comment below is not accurate as it says heating programs.
Either fix the comment or I suggest to actually rename it like suggested

Reply:
In ViCare context there exists a heating program (which can be heating and cooling) and a ventilation program. So to avoid confusions with ventilation in the future, I will rename it back to heating program.